src/site/models/ticketpurchase.php

```diff
@@ -395,6 +396,9 @@ private function memberAllowedToViewBuyTicket($member, $ticket)
 		{
 			$reason = "You need to be SWA Committee to buy this ticket.";
 		}
+		elseif (isset($t->details->first_event) && $member->num_tickets > 0) {
+			$reason = "You may only buy this ticket if it is your first event.";
+		}
```

Comment:
This should go up with the other if statements and have a `if ($t->details->visible == "Match" && !$member->swa_committee)` check. Otherwise it breaks `"visible": "Match"` in the ticket details JSON.

Reply:
It looks like the committee and qualification flags also ignore "visible":"Match". Is this deliberate? So that people can see that they need to upload a qualification? Still I would have though that you would just set the visibility differently in that case.